src/main/java/text/Print.java

```diff
@@ -0,0 +1,42 @@
+package text;
+
+import task.TaskManager;
+
+public class Print {
```

Comment:
![image](https://user-images.githubusercontent.com/35750468/93706684-3059ab00-fb5b-11ea-8362-e84213165b25.png)

take note of your naming!